components/ChatFooter.vue

```diff
@@ -275,6 +283,7 @@ import { fetchOurOffers } from '../composables/useThrottle'
 import { useAuthStore } from '../stores/auth'
 import { useAddressStore } from '../stores/address'
 import { untwem } from '~/composables/useTwem'
+import SpinButton from "./SpinButton";
```

Comment:
This line gives a lint error for me.  My IDE (PhpStorm) is set to show lint errors and autofix.  Can you set up to do the same?

There are other examples of this, and it would be nice if your PRs didn't introduce lint errors.